README.md

```diff
@@ -1,264 +1,505 @@
+# NAMETBD: A design system integrating to Pattern Lab and a Drupal 8 theme
+
 [![Build Status](https://travis-ci.org/phase2/pattern-lab-starter.svg?branch=master)](https://travis-ci.org/phase2/pattern-lab-starter)
 
-# Prerequisites
+NAMETBD is an opinionated set of tools and examples to:
+
+1. Build an application-agnostic **design system**
+1. Apply that design system to a locally-served **Pattern Lab** for rapid prototyping
+1. Apply that design system to a **Drupal theme**
+
+In depth documentation about frontend approach using this project at [Phase2 Frontend Docs](https://phase2.github.io/frontend-docs/)
+
+## Prerequisites
+
+- [Node `^8.0.0`](https://nodejs.org)
+- [PHP `5.6`, `^7.0.0`](https://php.net)
+- [Composer `^1.0.0`](https://getcomposer.org)
 
-- [Node](https://nodejs.org) v6, v8 is required for pa11y testing
-- PHP 5.4, 5.5, 5.6, OR 7 (5.3 might work, no promises)
-- [`composer`](https://getcomposer.org)
+## Provides
 
-# QuickStart
+- Drupal theme and Pattern Lab app
+- Strict [Atomic Design](http://atomicdesign.bradfrost.com/) component structure
+- Webpack bundling of all CSS, javascript, font, and static image assets for multiple targets (Drupal theme and Pattern Lab)
+- Webpack Dev Server for local hosting and auto asset injection into Pattern Lab and Drupal
+- Auto namespace addition into Drupal theme and Pattern Lab. Within any twig file, `@atoms/thing.twig` means the same thing to Drupal theme and Pattern Lab.
+- Iconfont auto-generation
+- Bootstrap 4 integration, used for all starting example components
+- Auto-linting against the [AirBnB JavaScript Style Guide](https://github.com/airbnb/javascript) and sane Sass standards
+- All Webpack and Gulp files are fully configurable
 
-[Download this repo](https://github.com/phase2/pattern-lab-starter/archive/master.zip), rename folder to `patternlab` and place in themes directory if integrating with Drupal 8, and then `cd` into folder before running below commands.
+## Quickstart
+
+NAMETBD can be run from anywhere to work with Pattern Lab. It also provides a theme to a Drupal website.
+
+### Quickstart anywhere
+
+1. [Download the latest release](https://github.com/phase2/pattern-lab-starter/releases)
+1. Extract anywhere (i.e. this readme should be at `any/where/NAMETBD/README.md`)
+1. Within the extracted folder run:
 
 ```bash
-npm install # or `yarn install` if you want 3x the speed
+npm install
 npm run setup
 npm start
 ```
 
-If you're using Drupal 8, get the [Component Libraries module](https://www.drupal.org/project/components):
+Simply wait until the webpack bundle output appears then visit [http://0.0.0.0/pl](http://0.0.0.0/pl) (or [http://localhost/pl](http://localhost/pl)) and start working.
```

Comment:
I think when this makes it to the FE docs, the "webpack bundle output" should have a further explanation/example of what this is/means.

Reply:
Given how new Webpack is to us, I think the FE docs around it will need to be quite detailed. You're right that it will get more thorough treatment over there (and hopefully an updated link from this point in this readme)